ui/src/notebooks/context/app.tsx

```diff
@@ -32,11 +36,20 @@ export const AppSettingContext = React.createContext<AppSettingContextType>(
 
 export const AppSettingProvider: FC<Props> = React.memo(
   ({timeZone, onSetTimeZone, children}) => {
+    const [miniMapVisibility, setMiniMapVisibility] = useState<boolean>(false)
```

Comment:
feel free to add to the app setting reducer here:
https://github.com/influxdata/influxdb/blob/master/ui/src/shared/reducers/app.ts

seems fitting